doc/installation.rst

```diff
@@ -1,65 +1,50 @@
 ..
-  SPDX-FileCopyrightText: 2016 - 2023 The Atlite Authors
+  SPDX-FileCopyrightText: Contributors to Atlite <https://github.com/pypsa/atlite>
 
   SPDX-License-Identifier: CC-BY-4.0
 
-############
+##############
 Installation
-############
+##############
 
-There are three possibilities to install atlite:
 
-* conda::
+Getting Python
+==============
 
-    conda install -c conda-forge atlite
+If it is your first time with Python, we recommend `conda
+<https://docs.conda.io/en/latest/miniconda.html>`_, `mamba
+<https://github.com/mamba-org/mamba>`_ or `pip
+<https://pip.pypa.io/en/stable/>`_ as easy-to-use package managers. They are
```

Comment:
```suggestion
`pip <https://pip.pypa.io/en/stable/>`_ can also be used, but may cause problems with some GIS related Python packages.
```

Reply:
Is this the case? Tests right now run on `pip`/ `uv`

Reply:
See here: https://conda.org/blog/2023-11-06-conda-23-10-0-release/

Reply:
No, I mean regarding `pip` which "may cause problems with some GIS related Python packages."